pixelpact/src/compare.test.js

```diff
@@ -0,0 +1,17 @@
+import fs from "fs/promises";
+import { compare } from "./compare.js";
+
+describe("compare", () => {
+  it("returns the number of pixels that differ", async () => {
+    let expected = await fs.readFile("testdata/simple-expected.png");
+    let actual = await fs.readFile("testdata/simple-actual.png");
+    let expectedDiff = await fs.readFile("testdata/simple-diff.png");
+
+    let result = await compare(expected, actual);
+
+    expect(result.numDiffPixels).toBe(152);
```

Comment:
Can we assign `152` to a variable with a name that describes its purpose, so that it `152` does not look like a magic number